config/schema/rules.schema.yml

```diff
@@ -10,18 +10,18 @@ rules.component.*:
     label:
       type: label
       label: 'Label'
-    module:
-      type: string
-      label: 'Module'
     description:
       type: text
       label: 'Administrative description'
-    tag:
-      type: string
-      label: 'Tag'
-    core:
-      type: string
-      label: 'Drupal version'
+    tags:
+      type: sequence
+      label: Tags
+      sequence:
+        type: string
+        label: 'Tag'
+    config_version:
+      type: float
```

Comment:
float? This is not really a floating number; if we go with "3.0" notation then I think this should be string